src/components/project_admin/suite_settings.vue

```diff
@@ -50,19 +48,52 @@
           </template>
         </select-object>
       </div>
-
       <div class="toggle-container form-field-wrapper">
-        <toggle v-model="d_suite.allow_network_access"
-                @input="$emit('field_change', d_suite)"
-                ref="allow_network_access">
+        <div>
+          <i class="fas fa-exclamation-triangle warning-tip"></i>
+          <tooltip width="large" placement="right">
+            Allowing network access may expose your files to students' code.
```

Comment:
Let's try this for the warning message:

Allowing network access means that student code could steal sensitive files or other information present in the sandbox (e.g., test cases, solution files) with a simple network request. If network access is required for your test cases to work, you may be able to mitigate the risk by deleting sensitive files at the and of the test suite setup phase. If there are sensitive files that cannot be deleted from the sandbox, then you should be aware of the risks associated with this setting.